CMakeLists.txt

```diff
@@ -97,7 +110,7 @@ set(GTEST_VERSION "1.11.0" CACHE STRING "googletest version in use" FORCE)
 
 set(OPAE_VERSION_LOCAL   "" CACHE STRING "OPAE local version")
 set(OPAE_VERSION_MAJOR    2 CACHE STRING "OPAE major version" FORCE)
-set(OPAE_VERSION_MINOR    2 CACHE STRING "OPAE minor version" FORCE)
+set(OPAE_VERSION_MINOR    3 CACHE STRING "OPAE minor version" FORCE)
```

Comment:
Be sure to reset the version to 2.2.0 after this goes in.  This change will set it to 2.3.0.

Reply:
fixed